packages/amplify-graphql-transformer-core/package.json

```diff
@@ -41,6 +41,7 @@
     "vm2": "^3.9.8"
   },
   "peerDependencies": {
+    "@aws-amplify/amplify-cli-core": "4.0.1",
```

Comment:
amplify-cli-core has been used in amplify-graphql-transformer-core without defining the dependency explicitly. In this PR, we are defining it explicitly so that the build tree resolves the dependency correctly.

Here is a previously merged and released PR using amplify-cli-core package's stateManager.
https://github.com/aws-amplify/amplify-category-api/pull/1372